about.html

```diff
@@ -0,0 +1,216 @@
+        </p>
+        <div class="logo-container">
+          <img src="images/logo.png" class="logo-border" alt="logo" />
+          <p class="contact-me">Feel free to contact us for more info:</p>
+        <div class="email">
+          <u>af.esthetique@gmail.com</u>
+        </div>
+        </div>
+      </section>
+      <section class="testimonials">
+        <h2 class="about-logo-title">Testimonials</h2>
+        <div class="bar"></div>
+        <p class="about-logo-info marg">
+          Take a look at what our clients are saying about Af Esthétique's
+          services
+        </p>
+        <div class="testimonials-container">
+          <div class="testimonials-box box1">
+            <img
+              class="testimonials-background"
+              src="images/sandrine.jpg"
+              alt="2020 Testimonial by Sandrine Lebon"
+            />
+            <h2 class="testimonials-title">Sandrine Lebon</h2>
+            <p class="testimonials-info">
+              Bibi Af is very professional. She makes you feel at ease and her
+              technique is great. She is really good at getting rid of those
+              knots.
+            </p>
+          </div>
+          <div class="testimonials-box box2">
+            <img
+              class="testimonials-background"
+              src="images/michelle.jpg"
+              alt="Testimonial by Isabelle Goetz"
+            />
+            <h2 class="testimonials-title">Isabelle Goetz</h2>
+            <p class="testimonials-info">
+              Home service is so convenient for me as I've just had a baby
+              recently. I looked online and found Af Esthétique. I give Bibi Af
+              a rating of 5 stars. She's the best I've known so far.
+            </p>
+          </div>
+        </div>
+      </section>
+      <footer>
+        <div class="customers-container">
+          <h2 class="customers">Satisfied Makeup Customers</h2>
+          <div class="bar customers-bar"></div>
+          <div class="customers-small-container">
+            <div class="customers-img-container">
+              <img
+                src="images/customer.jpg"
+                class="customers-img"
+                alt="Satisfied customer 1"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer2.jpg"
+                class="customers-img"
+                alt="Satisfied customer 2"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer3.jpg"
+                class="customers-img"
+                alt="Satisfied customer 3"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer4.jpg"
+                class="customers-img"
+                alt="Satisfied customer 4"
+              />
```

Comment:
- [ ] There is no `satisfied makeup customer` or `partner` section in the `desktop version` of `about.html`.

### Desktop version

### See Behance design 

![Screenshot from 2021-07-09 20-51-55](https://user-images.githubusercontent.com/22434660/125102475-f2813f80-e0f8-11eb-8717-a43044c9b0ce.png)


### Your design

![Screenshot from 2021-07-09 20-52-53](https://user-images.githubusercontent.com/22434660/125102461-ee552200-e0f8-11eb-94b7-f03db6c2d65d.png)